CMakeLists.txt

```diff
@@ -155,9 +155,15 @@ endif()
 ### Wave components [WW3]
 ###############################################################################
 if(WW3)
-  # Set options for WW3 build
-  set(SWITCH "multi_esmf" CACHE STRING "Default to switch_multi_esmf")
-  set(MULTI_ESMF ON CACHE BOOL "Enable ESMF for WW3")
+  if()
```

Comment:
how do we get to this if versus the other if ? 
Would there be something specified at compile time?

Reply:
Thanks for the explanation, so essentially this is a placeholder for what would be the if statement.

Reply:
Would I be able to use the app name here? Since only the ATMW app will remain using connectors/existing cap, something like...

if(ATMW)
 --> multi_esmf cap
else
 -> nuopc_mesh cap
endif

Reply:
Yes, this is just a placeholder for the real logic.

@DeniseWorthen you can use the app name there, like `if(APP MATCHES "^(ATM)$")`